src/apps/wallet/sign_tx/layout.py

```diff
@@ -65,3 +84,16 @@ def split_op_return(data):
     text = Text("Confirm sending", ui.ICON_SEND, icon_color=ui.RED)
     text.normal("Trying to spend", "coins from another chain.", "Continue?")
     return await confirm(ctx, text, ButtonRequestType.SignTx)
+
+
+async def confirm_nondefault_locktime(ctx, lock_time):
+    text = Text("Confirm locktime", ui.ICON_SEND, icon_color=ui.GREEN)
+    if lock_time < 500000000:
```

Comment:
Let's extract `500000000` as a constant similarly as `_EMBEDDED_TIMESTAMP_DIFF` is extracted.

Reply:
done